### PR TITLE
Update user map with institutions added in April 2016 #5250

### LINKED ISSUE #5250: Update user map with institutions added in April 2016
Can follow #5163
Here's the list for this month
```
King's Baptist Grammar School	Australia
University of Sydney	Australia
Morling College	Australia
Morling College	Australia
Narangba Valley SHS	Australia
BA Stassart	Belgium
Arteveldehogeschool	Belgium
Artevelde University College Ghent	Belgium
Southern Alberta Institute of Technology	Canada
University of Lethbridge	Canada
University of Windsor	Canada
St. Mary's High School	Canada
Langara College	Canada
University of Guelph	Canada
Royal Roads University	Canada
Universidad De Chile	Chile
Universidad EAFIT 	Colombia
IESEG School of Management	France
CentraleSupélec	France
P S Senior Secondary School	India
Cork Institute of Technology	Ireland
Universiti Malaysia Sarawak	Malaysia
Instituto Tecnológico y de Estudios Superiores de Monterrey	Mexico
Summa College	Netherlands
Summa College	Netherlands
Avans Hogeschool	Netherlands
Utrecht University	Netherlands
TU Delft	Netherlands
Saint Louis University	Philippines
Ngee Ann Polytechnic	Singapore
SUTD	SIngapore
Singapore Institute of Technology	Singapore
Changkat Changi Secondary School	Singapore
Temasek Polytechnic	Singapore
Gimnazija Celje - Center	Slovenia
University of Navarra	Spain
China Institute of Technology and Science	Taiwan
ALHOSN University	UAE
University of Manchester	UK
Falmouth University	UK
University of Salford	UK
University of Salford	UK
University of Bristol	UK
Chicago State University	USA
Indiana State University	USA
Whatcom Community College	USA
Creighton University	USA
California Baptist University	USA
Northwest High School	USA
Unversity of Virginia	USA
Northeastern University	USA
University of San Diego	USA
NC A&T State University	USA
UCLA	USA
New Mexico State University	USA
Juan Seguin High School	USA
Burrell College of Osteopathic Medicine	USA
Burrell College Osteopathic Medicine	USA
Shaw High School	USA
Ohio University	USA
Savannah Arts Academy	USA
Sam Houston State University	USA
Texas Tech University	USA
Edward W. Clark High School	USA
University of Virginia	USA
University of San Diego	USA
California State University, Chico	USA
Gaston College	USA
Purdue University	USA
Purdue University	USA
Milwaukee Area Technical College	USA
University of South Carolina	USA
University of Texas at Arlington	USA
Southern Methodist University	USA
Iowa State University	USA
Mercer University	USA
El Paso Community College	USA
Duke University	USA
Ferris State University	USA
Colorado State University	USA
Chester County High School	USA
Otterbein University	USA
Lamar Consolidated Independent School District	USA
Chamberlain College of Nursing	USA
Chamberlain College of Nursing	USA
University of Colorado Boulder	USA
University of Georgia	USA
University of San Diego	USA
University of Massachusetts-Amherst	USA
South Carolina State University	USA
Lincoln-King Academy	USA
Mercer Middle School	USA
South Plains College	USA
Syracuse University	USA
Rensselaer Polytechnic Institute	USA
University of South Dakota	USA
University at Albany	USA
South Paulding High School	USA
University of California Los Angeles	USA
Virginia Commonwealth University	USA
University of Arizona	USA
University of Colorado Boulder	USA
Bowling Green State University 	USA 
```

### PR DESCRIPTION
**Pull request by mariasam1**
Fixes #5250

Added the institutions above. Some of the colleges in the original issue post appeared twice:
University of Salford   UK, Summa College Netherlands, UCLA, Morling College, Burrell College Osteopathic Medicine    USA, Chamberlain College of Nursing  USA, Purdue University   USA, University of Colorado Boulder  USA
And a few were already on the original list of institutions.

### CODE REVIEW

**Comment by chowyb** [comment]
Hi @mariasam1,

You may want to include the original issue number in the PR description.

Meanwhile, I'll have a look through the changes to see if everything is okay.

**Comment by chowyb** [comment]
Changes look good, waiting on PR description update now.

File: src/main/webapp/js/userMapData.json [hunk 1]
```diff
@@ -207,12 +222,14 @@
 ["University Kuala Lumpur","Malaysia"],
 ["University of Nottingham Malaysia Campus","Malaysia"],
 ["Escuela Bancaria y Comercial","Mexico"],
+["Instituto Tecnológico y de Estudios Superiores de Monterrey","Mexico"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: This is same as ITESM in the next line. Should be removed.

File: src/main/webapp/js/userMapData.json [hunk 2]
```diff
@@ -585,6 +625,7 @@
 ["N2N Services","USA"],
 ["Nash Rocky Mount Public Schools","USA"],
 ["Navarro College","USA"],
+["NC A&T State University","USA"],
 ["NC State University","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Check if these two are different.

File: src/main/webapp/js/userMapData.json [hunk 3]
```diff
@@ -752,7 +805,9 @@
 ["University of Saint Mary","USA"],
 ["University of San Diego","USA"],
 ["University of Scranton","USA"],
+["University of South Carolina","USA"],
 ["University of South Carolina Aiken","USA"],
```
Comment (damithc) [inline src/main/webapp/js/userMapData.json]: Check if these two are different.

File: src/main/webapp/js/userMapData.json (hunk 1 above)
Comment (mariasam1) [inline src/main/webapp/js/userMapData.json]: OK, removed.

File: src/main/webapp/js/userMapData.json (hunk 2 above)
Comment (mariasam1) [inline src/main/webapp/js/userMapData.json]: These are different - North Carolina Agricultural and Technical State University is in Greensboro, and NC State University is in Raleigh.

File: src/main/webapp/js/userMapData.json (hunk 3 above)
Comment (mariasam1) [inline src/main/webapp/js/userMapData.json]: These are different - University of South Carolina is in Columbia, and University of South Carolina Aiken is in Aiken.

**Comment by chowyb** [comment]
@mariasam1 no need for the dash in the PR title, apart from that everything looks good now :+1:

**Comment by chowyb** [comment]
@mariasam1 alright looks good to go!

**Comment by chowyb** [comment]
Hmm actually come to think of it, I'm not sure if setting the last updated date to 4 July is a good idea, since we still have another earlier issue (#5538) for the user map.

@damithc any suggestions? Setting the date to today would be a bit odd since several institutions are not reflected yet, but setting the date to April may seem a bit odd as well, since the changes were made only recently.

**Comment by damithc** [comment]
Good point. The date should be the end of the month in concern. e.g.  April 31st.

**Comment by mariasam1** [comment]
Sounds good - I committed the new date. I can work on #5538 too.

**Comment by chowyb** [comment]
Just a minor change, but we should probably change the date to 30 April; 31 April doesn't exist =P

**Comment by mariasam1** [comment]
Oops, changed it. 👍

**Comment by mariasam1** [comment]
Never mind, guess I can't work on #5538 because only "one contributor/developer is limited to one such issue" :/ https://github.com/TEAMMATES/teammates/pull/5234

**Comment by chowyb** [comment]
Okay, good to go :+1:

**Pull request merged**